src/analysis/normed_space/basic.lean

```diff
@@ -1255,6 +1255,15 @@ namespace real
 lemma norm_of_nonneg {x : ℝ} (hx : 0 ≤ x) : ∥x∥ = x :=
 abs_of_nonneg hx
 
+lemma norm_of_nonpos {x : ℝ} (hx : x ≤ 0) : ∥x∥ = -x :=
+abs_of_nonpos hx
+
+lemma norm_of_pos {x : ℝ} (hx : 0 < x) : ∥x∥ = x :=
```

Comment:
I don' think this one is needed, as one can always use `norm_of_nonneg h.le` when one has an assumption `h : 0 < x`. Same thing for the next one.